python/requirements.txt

```diff
@@ -8,3 +8,4 @@ packaging>=17.0
 numpy~=1.11
 PyYAML~=6.0.1
 pyDOE~=0.3.8
+DateTime~=5.0
```

Comment:
I have nothing against this, but I do not see where it is used. Can you tell it to me, please? 😅

Reply:
I do not remember why I put this requirement. I think I had some problem on my office machine. In any way, this problem does not occur on my laptop, and "DateTime" (in contrast to "datetime") is not used, so I removed this again.